src/muqsit/fakeplayer/DefaultFakePlayerListener.php

```diff
@@ -70,15 +72,14 @@ public function onPlayerAdd(Player $player) : void{
 				if($session->isConnected()){
 					$player = $session->getPlayer();
 					if($player !== null){
-						$position = $player->getPosition()->floor();
+						$vector3 = $player->getPosition()->floor();
 
-						$packet = new PlayerActionPacket();
-						$packet->action = PlayerActionPacket::ACTION_DIMENSION_CHANGE_ACK;
-						$packet->entityRuntimeId = $player->getId();
-						$packet->face = 0;
-						$packet->x = $position->x;
-						$packet->y = $position->y;
-						$packet->z = $position->z;
+						$packet = PlayerActionPacket::create(
+							$player->getId(),
+							PlayerAction::DIMENSION_CHANGE_ACK,
+							BlockPosition::fromVector3($vector3),
```

Comment:
May as well inline $vector3 while at it, since the variable's name isn't descriptive of it's usage
```diff
- $vector3 = $player->getPosition()->floor();

- BlockPosition::fromVector3($vector3),
+ BlockPosition::fromVector3($player->getPosition()->floor()),
```